### DISCUSSION TITLE
May I use my GitHub Copilot subscription with this project?

### DISCUSSION THREAD

**Discussion by b9Joker108** [yetone/avante.nvim#745] [Q&A]
I am pretty much a coding noob, though I have been coding for a few years: mostly shell and Python scripts. I apologise in advance for my stoopid questions. I have subscribed to a year of GitHub copilot, which I have not yet got into the swing of using regularly. May I use my current GitHub subscription to furnish all the requirements and functionlities of avante.nvim? I noted reading discussions and issues, that there was an unfair usage or abuse of GitHub Copilot due to planery.curl or something, sent as correspondence to a user of this project. So, I am wondering if this precludes the usage of GitHub Copilot for this project?

**Comment by HadyMash** [comment]
You can use the GitHub copilot, but you need to have logged into Copilot.lua first (by running `:Copilot auth`). For suggestions, I've also seen some posts about issues with the autosuggestion. You can use Copilot.lua for autocomplete instead of avante, and use avante for chat/code gen/editing.
